### PR TITLE
HToAATo4B production cards: WH

### PR DESCRIPTION
**Pull request by abrinke1**
Generation script, template cards, and cards for all mass points for WH.
See https://github.com/cms-sw/genproductions/pull/2880

### CODE REVIEW

**Comment by abrinke1** [comment]
Hi @sihyunjeon , @agrohsje ,

I have updated the WH production with the fixes from ttH (https://github.com/cms-sw/genproductions/pull/2903).  The run_card is identical to ttH.  Please let me know if there are any other settings that need to be adjusted.

Thanks much,
@abrinke1

**Comment by agrohsje** [comment]
Looks good to me. @sihyunjeon feel free to merge if you are as well ok.

**Comment by abrinke1** [comment]
Hi @sihyunjeon , I'm guessing this just slipped through the cracks?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_WH_WToAll_HToAATo4B_M-MASS/SUSY_WH_WToAll_HToAATo4B_M-MASS_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import model NMSSM --modelname
+
+generate p p  > w+ h01 , (h01 > a01 a01, a01 > b b~, a01 > b b~)
+generate p p  > w- h01 , (h01 > a01 a01, a01 > b b~, a01 > b b~)
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_WH_WToAll_HToAATo4B_M-MASS/SUSY_WH_WToAll_HToAATo4B_M-MASS_proc_card.dat:4]: Here, it should be "add process".
Can you double check for all the other requests you made that this didn't happen? I remember I found one for you but did not look closely enough to the other ones

Comment (abrinke1) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_WH_WToAll_HToAATo4B_M-MASS/SUSY_WH_WToAll_HToAATo4B_M-MASS_proc_card.dat]: Hi @sihyunjeon ,
Good catch again!  I've fixed it.  WH and ggH+0/1 jet are the only cards with two processes.

**Comment by abrinke1** [comment]
Hi @sihyunjeon , @agrohsje ,
Just a quick reminder to merge this last one.
Thanks!
Andrew

**Comment by sihyunjeon** [comment]
merged

**Pull request merged**